README.md

```diff
@@ -290,7 +290,63 @@ Key | Description | Possible Values
 
 For Jira you can also specify custom fields that will be populated with values.
 Use the `unknowns` parameter in cfg.yaml for custom fields.
-Under the `unknowns` parameter, specify the list of fields names to provide value for.
+Under the `unknowns` parameter, specify the list of fields **names** to provide value for. Field name can contains spaces.
+  
+Possible options for getting the field name:
+
+<details> <summary> Get field name from Jira UI </summary>
+
+1. Move to your jira.
+2. Navigate to **Settings**(![cog](https://user-images.githubusercontent.com/91113035/159643662-b7a21717-58f0-4a5e-87a0-0d840046e215.png)) > **Issues** > **Custom fields** under the Fields section:
+![Снимок экрана от 2022-03-23 13-17-48](https://user-images.githubusercontent.com/91113035/159644478-cb3d880f-0a2d-4e95-b54f-600b753e0b12.png)
+3. Click on the required field.
+  
+![Снимок экрана от 2022-03-23 13-19-07](https://user-images.githubusercontent.com/91113035/159646058-1e8e25c2-fb65-46e7-83e9-e439ef447904.png)
```

Comment:
Can we add these images to the repo themselves? I'm not sure if relying on githubuserconent is reliable.

Reply:
And also if you could change the markdown tags for the images to be in English that'll be great.

Reply:
Done

Reply:
thanks